packages/trader/src/App/Components/Elements/ContractAudit/contract-details.tsx

```diff
@@ -95,6 +97,30 @@ const ContractDetails = ({
         ''
     );
 
+    const createLookBacksMarker = (abbreviation?: string) => {
+        let label: string;
+
+        switch (abbreviation) {
+            case 'L':
+                label = localize('Indicative low spot');
+                break;
+            case 'H':
+            default:
+                label = localize('Indicative high spot');
+        }
+
+        return {
+            label,
```

Comment:
```suggestion
            label: abbreviation === 'L' ? localize('Indicative low spot') : localize('Indicative high spot'),
```
If there are only 2 options for `abbreviation` that can be possibly received here (`H` and `L`), Can we use ternary instead of the big `switch case` above? :)

Reply:
<Localize /> component?

Reply:
Sure